### PR TITLE
update demo data and add mac getter for ssh

### PR DESCRIPTION
**Pull request by itdependsnetworks**
<!-- Make sure you have read http://napalm.readthedocs.io/en/latest/contributing/index.html --!>
Mac getter seems to be good, and grabbed some demo data from Guido and just searching github.

### CODE REVIEW

File: napalm_nxos_ssh/nxos_ssh.py [hunk 1]
```diff
@@ -672,11 +675,105 @@ def get_interfaces_ip(self):
...
+        * 27       001b.54c2.2644    dynamic      -       F    F    po1
+        * 27       0000.0c9f.f2bc    dynamic      -       F    F    po1
+        * 27       0026.980a.df44    dynamic      -       F    F    po1
+        * 16       0050.56bb.0164    dynamic      -       F    F    po2
+        * 13       90e2.ba5a.9f30    dynamic      -       F    F    eth1/2
+        * 13       90e2.ba4b.fc78    dynamic      -       F    F    eth1/1
+        """
+
+        #  The '*' is stripped out later
+        RE_MACTABLE_FORMAT1 = r"^\s+{}\s+{}\s+\S+\s+\S+\s+\S+\s+\S+\s+\S+".format(
```
Comment (mirceaulinic) [inline napalm_nxos_ssh/nxos_ssh.py:707]: As this regex is used inside a loop, I'd suggest to pre-compile it.

**Review by mirceaulinic — APPROVED** [review]
Looking good! :-)

File: napalm_nxos_ssh/nxos_ssh.py (hunk 1 above)
Comment (ktbyers) [reply napalm_nxos_ssh/nxos_ssh.py:707]: @mirceaulinic it doesn't matter...Python automatically compiles and caches the results on the first use. So we are talking about very insignificant performance differences.

Comment (mirceaulinic) [reply napalm_nxos_ssh/nxos_ssh.py:707]: Yeah, my bad - it's in my nature to try to optimise things and make them run fast :-|

**Pull request merged**